src/components/GuardianCard.tsx

```diff
@@ -0,0 +1,61 @@
+import { Gray1, Gray3, Gray4, TextDark, TextLight } from '../colors';
+
+import React from 'react';
+import { rem } from 'polished';
+import styled from 'styled-components';
+
+type Props = {
+  address: string;
+  status: number;
+};
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  /* justify-content: center; */
+  align-items: center;
+  border: 1px solid ${Gray3};
+  width: ${rem(240)};
+  height: ${rem(230)};
+  border-radius: ${rem(16)};
+  background: linear-gradient(to bottom, ${Gray1} 0%, ${Gray1} 64px, #FFFFFF 64px, #FFFFFF 100%);
+`;
+
+const Ellipse = styled.div`
+  border-radius: 50%;
+  background: #EBE4FF;
```

Comment:
is there a color match in the `../colors.ts` file? if not, maybe we can create one

Reply:
Not yet that I could find, ill add it, thanks :)